api/globalv1/api.proto

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package globalv1;
+
+import "api/localv1/api.proto";
+
+option go_package = "sigs.k8s.io/kpng/api/globalv1";
+
+//
+// global (cluster-wide) model
+//
+
+message ServiceInfo {
```

Comment:
Kind of a Nit but I think we should add comments to all of these types just like we do in K8s, It might be a bit tedious but may be helpful for folks trying to understand it 

(we also could tackle it in a follow up PR)

Reply:
agree we need to comment them all .  actually this entire pr stems from the fact that me and amim couldn't figure out what any of these structs where differentiated for .... so we got mikael to split up the files. 

lets do that in a follow